tests/ebpf_xdp.h

```diff
@@ -1,25 +1,27 @@
-#ifndef __BPF_HELPERS_H
-#define __BPF_HELPERS_H
+/*
+Copyright 2018 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
 
-/* Additional headers */
-# define printk(fmt, ...)                                               \
-                ({                                                      \
-                        char ____fmt[] = fmt;                           \
-                        bpf_trace_printk(____fmt, sizeof(____fmt),      \
-                                     ##__VA_ARGS__);                    \
-                })
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
 
+/*
+ * This file contains all functions and definitions necessary for the xdp target C code to compile. It must be included with any file generated by the p4c-xdp compiler.
+ */
+
+#ifndef __BPF_HELPERS_H
+#define __BPF_HELPERS_H
 
-#include <stdbool.h>
-#include <stdio.h>
-typedef signed char s8;
-typedef unsigned char u8;
-typedef signed short s16;
-typedef unsigned short u16;
-typedef signed int s32;
-typedef unsigned int u32;
-typedef signed long long s64;
-typedef unsigned long long u64;
+#include "ebpf_kernel.h"
```

Comment:
This adds a dependency on the kernel target. Not sure if we want to do that or just duplicate all the content in the kernel header.